### ISSUE TITLE
DOC: add example of to .dask (HighLevelGraph repr) to docs in https://docs.dask.org/en/latest/graphs.html

### ISSUE THREAD

**Issue by raybellwaves** [dask/dask#8443]
Only learned about `.dask` after @charlesbluca's dask-sql demo at the dask meeting. Not sure if it's in the docs anywhere?

Could add a Visualization subpage to https://docs.dask.org/en/latest/graphs.html ?

**Comment by mrocklin** [comment]
cc @GenevieveBuckley @freyam

**Comment by freyam** [comment]
Yes, this would be a great addition.

I have my final exams going on right now. I will be working on #7709 starting 2 weeks from now.

I will make sure to add documentation about `.dask` as well.

**Comment by raybellwaves** [comment]
Didn't see #7709. Welcome to close this if you think this is a dup

**Comment by GenevieveBuckley** [comment]
Yes, we should do this. If no-one knows it exists, it might as well not exist. I'm a bit busy, but I'll try and squeeze something in, you're right that it is important. 

Strictly speaking, I don't think this is a duplicate of https://github.com/dask/dask/issues/7709. Let's keep both issues open, they seem to address slightly different aspects of the docs.

**Comment by GenevieveBuckley** [comment]
The other thing I think is not widely known is the `result.dask.visualize()` method. Everyone knows `result.visualize()` (low level graph visualization) but not the high level graph one. 

That should also get added to the docs in a more prominent location. Like near where we introduce the first `.visualize()` method.

Maybe these things should also go in the tutorials at some stage too?

**Comment by raybellwaves** [comment]
Indeed I did not know about `result.dask.visualize()` thanks

**Comment by GenevieveBuckley** [comment]
I feel like this page is probably the better place for it: https://docs.dask.org/en/latest/graphviz.html

Opening a PR now
